<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoviePlayer</class>
 <widget class="QMainWindow" name="MoviePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Movie Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutPrincipal"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionCapturarVideo"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionAbrirCon"/>
    <addaction name="separator"/>
    <addaction name="actionAbrirDirectorio"/>
    <addaction name="actionAbrirTerminal"/>
    <addaction name="separator"/>
    <addaction name="actionRenombrar"/>
    <addaction name="actionEliminar"/>
   </widget>
   <widget class="QMenu" name="menuReproduccion">
    <property name="title">
     <string>Reproducción</string>
    </property>
    <widget class="QMenu" name="menuVelocidad">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Velocidad</string>
     </property>
     <addaction name="actionNormal"/>
     <addaction name="separator"/>
     <addaction name="actionIncrementar"/>
     <addaction name="actionReducir"/>
     <addaction name="separator"/>
     <addaction name="actionDoblar"/>
     <addaction name="actionMitad"/>
    </widget>
    <addaction name="actionReproducir"/>
    <addaction name="actionPausar"/>
    <addaction name="actionDetener"/>
    <addaction name="separator"/>
    <addaction name="actionAvanzar"/>
    <addaction name="actionRetroceder"/>
    <addaction name="separator"/>
    <addaction name="actionSiguienteFotograma"/>
    <addaction name="actionAnteriorFotograma"/>
    <addaction name="separator"/>
    <addaction name="menuVelocidad"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionCapturarPantalla"/>
    <addaction name="separator"/>
    <addaction name="actionAjustarVentana"/>
    <addaction name="actionActivarCache"/>
   </widget>
   <widget class="QMenu" name="menuPreferencias">
    <property name="title">
     <string>Preferencias</string>
    </property>
    <addaction name="actionDispositivos"/>
    <addaction name="actionAutoReproducir"/>
    <addaction name="actionPantallaCompleta"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="separator"/>
    <addaction name="actionAcercaDe"/>
    <addaction name="actionAcercaDeQt"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuReproduccion"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuPreferencias"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarSuperior">
   <property name="windowTitle">
    <string>Barra de navegación</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="actionCapturarVideo"/>
   <addaction name="actionCerrar"/>
  </widget>
  <widget class="QToolBar" name="toolBarInferior">
   <property name="windowTitle">
    <string>Barra de reprodución</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReproducir"/>
   <addaction name="actionPausar"/>
   <addaction name="actionDetener"/>
   <addaction name="actionRetroceder"/>
   <addaction name="actionAvanzar"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbrirRecientes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abrir recientes</string>
   </property>
  </action>
  <action name="actionGuardarComo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Guardar como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbrirCon">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abrir con...</string>
   </property>
  </action>
  <action name="actionAbrirDirectorio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abrir directorio</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAbrirTerminal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abrir terminal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionRenombrar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Renombrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAcercaDe">
   <property name="text">
    <string>Acerca de</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionAcercaDeQt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionReproducir">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reproducir</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionPausar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionDetener">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Detener</string>
   </property>
   <property name="shortcut">
    <string>Shift+Return</string>
   </property>
  </action>
  <action name="actionPantallaCompleta">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pantalla completa</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAjustarVentana">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ajustar a ventana</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionActivarCache">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activar caché</string>
   </property>
   <property name="statusTip">
    <string>Aviso: Alto consumo de memoria (por defecto archivos &lt;= 10MB)</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionIncrementar">
   <property name="text">
    <string>Incrementar</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionReducir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reducir</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionDoblar">
   <property name="text">
    <string>Doblar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionMitad">
   <property name="text">
    <string>Mitad</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSiguienteFotograma">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Siguiente fotograma</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAnteriorFotograma">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Anterior fotograma</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionAvanzar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Avanzar 1s</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionRetroceder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Retroceder 1s</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionCapturarPantalla">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tomar instantánea</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAutoReproducir">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto-reproducir</string>
   </property>
  </action>
  <action name="actionCapturarVideo">
   <property name="text">
    <string>Capturar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDispositivos">
   <property name="text">
    <string>Dispositivos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
